pwnlib/constants/linux/s390.py

```diff
@@ -511,7 +511,7 @@
 _SIGCONTEXT_NSIG = Constant('_SIGCONTEXT_NSIG',64)
 _SIGCONTEXT_NSIG_BPW = Constant('_SIGCONTEXT_NSIG_BPW',64)
 __SIGNAL_FRAMESIZE = Constant('__SIGNAL_FRAMESIZE',160)
-_SIGCONTEXT_NSIG_WORDS = Constant('_SIGCONTEXT_NSIG_WORDS',(64 / 64))
+_SIGCONTEXT_NSIG_WORDS = Constant('_SIGCONTEXT_NSIG_WORDS',(64 // 64))
```

Comment:
This is auto-generated from C source.  Please revert all changes to pwnlib/constants.